app/templates/base.html

```diff
@@ -109,7 +109,7 @@
                             {% if h_hw_enabled %}
                                 {% include 'include/hardware_tab.html' %}
                             {% endif %}
-                            {% if request.user.is_organizer or request.user.is_volunteer %}
+                            {% if request.user.check_is_organizer or request.user.check_is_volunteer_accepted %}
```

Comment:
Why rename this?

Reply:
volunteer must change to is_volunteer_accepted, the others i'll unchange them